dashboard-api/cmd/api/bootstrap/bootstrap.go

```diff
@@ -0,0 +1,39 @@
+package bootstrap
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+
+	"github.com/kelseyhightower/envconfig"
+	_ "github.com/lib/pq"
```

Comment:
Let's use pgx, which is what we're using everywhere else, unless there's an interesting reason to have a different driver?

Reply:
I was not aware that we were using pgx. The main reason I used pq it's because it supported database/sql but it seems like pgx also supports it, so I'm happy to change the dependency. 👌

Relying on the database/sql package from the go standard library for the repositories code it's good because it provides a generic interface for any SQL database, so updating repositories implementations becomes much easier.

Pgx can be used as standalone or in combination with database/sql, I prefer the second approach for what I just mentioned above. If at some point (or you already anticipate) we want some of the additional features that the standalone pgx version offers, we can migrate the repositories implementation.